libs/viewer/include/viewer/TweakableProperty.h

```diff
@@ -174,6 +174,21 @@ struct TweakableProperty {
             return mFilename;
         }
 
+        std::string getFileExtension() const {
+            std::string result = "";
+
+            size_t lastDotPlusOne = mFilename.find_last_of(".") + 1;
+            if (lastDotPlusOne < mFilename.length()) {
+                result = mFilename.substr(lastDotPlusOne);
```

Comment:
could use `utils::Path::getExtension()`

Reply:
Fixed in https://github.com/shapr3d/filament/pull/20/commits/ddf94a75580b2a0573fd9d18dc5bc78c44e25463